pose_graph/CMakeLists.txt

```diff
@@ -16,8 +16,8 @@ add_subdirectory(src/ThirdParty)
 
 # -------- Add library --------
 add_library(pose_graph STATIC
+    src/details/keyframe.cpp
```

Comment:
Just for my understanding, what is the rationale for having the details folder? Is that a common practise?